.github/CONTRIBUTING.md

```diff
@@ -10,10 +10,14 @@ This site is built with [Primer React components][components] and [x0].
 To start up your local dev environment, run:
 
 1. `npm install` to install all the Node dependencies
-2. `npm run dev` to start the dev server
-3. `npm run build` to build the static site
-4. `npm start` to serve the static site
+2. `npm start` to run the app in development mode
 
+Additionally, you can run `npm run now-build` to build the static site, and `npm run now-start` to run the static site on your local environment. This can be helpful for debugging build issues.
+
+We use the [React configuration](https://github.com/github/eslint-plugin-github/blob/master/lib/configs/react.js) from [GitHub's eslint plugin](https://github.com/github/eslint-plugin-github) to lint our JavaScript. To check your work before pushing, run:
+```
+npm run lint
+```
```

Comment:
@emplums I've added a small section about linting replicating some content from https://github.com/primer/components/commit/66176dd41f98a80a2e00c7ea9a292c30c58a4937#diff-cc4aac3e9be04e0413c9520f223b493cR14 since this repository is _also_ using react configuration from the eslint plugin in [`.eslintrc.json`](https://github.com/primer/components/blob/e29890b8558d2bfdc572e67f1816ba57ce4fcadf/.eslintrc.json#L4).